cobbler/actions/hardlink.py

```diff
@@ -28,7 +29,7 @@ def __init__(self, api: "CobblerAPI") -> None:
 
         :param api: The API to resolve information with.
         """
-        if api is None:
+        if api is None:  # type: ignore[reportUnnecessaryComparison]
```

Comment:
I think since it is unlikely that `api` is `None` and we enforce pyright, wouldn't it be an option to remove the check altogether? Grepping through the code it shows only two usages: The API itself and the tests.

Reply:
+1, let's remove the check.